packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -154,6 +177,50 @@ export class ModalBase extends BaseComponent<IModalProps, IDialogState> implemen
       className: classNames.layer
     };
 
+    const modalContent = (
+      <FocusTrapZone
+        componentRef={this._focusTrapZone}
+        className={classNames.main}
+        elementToFocusOnDismiss={elementToFocusOnDismiss}
+        isClickableOutsideFocusTrap={isModeless || isClickableOutsideFocusTrap || !isBlocking}
+        ignoreExternalFocusing={ignoreExternalFocusing}
+        forceFocusInsideTrap={isModeless ? !isModeless : forceFocusInsideTrap}
+        firstFocusableSelector={firstFocusableSelector}
+        focusPreviouslyFocusedInnerElement={true}
+        onKeyDown={isDraggable ? this._onDialogKeyDown : undefined}
+        onKeyUp={isDraggable ? this._onDialogKeyUp : undefined}
+        onBlur={isInKeyboardMoveMode ? this._onExitKeyboardMoveMode : undefined}
+      >
+        {isInKeyboardMoveMode && (
+          <div className={classNames.keyboardMoveIconContainer}>
+            {keyboardMoveIconProps ? <Icon {...keyboardMoveIconProps} /> : <Icon iconName="move" className={classNames.keyboardMoveIcon} />}
+          </div>
+        )}
+        <div ref={this._allowScrollOnModal} className={classNames.scrollableContent} data-is-scrollable={true}>
+          {isDraggable && this.state.isModalMenuOpen && (
+            <ContextualMenu
```

Comment:
Can we avoid adding this to Modal?

Reply:
The reason is that ContextualMenu adds menu code, adds positioning logic, pulls in FocusZone, etc... pulls in other things as well. All adds up.